src/generic/datavgen.cpp

```diff
@@ -631,6 +633,101 @@ class wxDataViewTreeNode
 
 WX_DEFINE_SORTED_ARRAY_SIZE_T(unsigned int, wxDataViewSelection);
 
+
+struct RangeEntity
+{
+    unsigned int from;
+    unsigned int to;
+};
+
+WX_DECLARE_OBJARRAY(RangeEntity, ArrayOfRangeEntities);
+#include "wx/arrimpl.cpp"
+WX_DEFINE_OBJARRAY(ArrayOfRangeEntities)
+
+class RowRange
```

Comment:
It's a bit confusing that this is called `RowRange` when it really seems to be a set of row ranges. I'd rename `RangeEntity` to `RowRange` (because this is what it is) and call this one `RowRanges`.